scripts/generate-python.sh

```diff
@@ -52,9 +52,9 @@ else
         rm protoc-3.15.8-linux-x86_64.zip
     fi
     ./.local/bin/protoc --version || exit 1
-    pip install --upgrade pip || exit 1
-    pip install mypy-protobuf==3.0 || exit 1
 fi
+pip3 install --upgrade pip || exit 1
```

Comment:
do these scripts need to be exposed in the external repo?

Reply:
Currently, yes, I think it's safer to copy them over.  GitHub Action configs are also copied over.  This helps make sure we can generate client libraries from the external repo commits.